package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "srp_js",
+  "version": "0.0.1",
```

Comment:
hmm... the Version file says 0.3.2 or so. The Readme states
```
The original repository had code working with a django server. This was removed after the commit tagged 0.1.0
```
The commit history and tags do not show these. No idea why. But maybe we can make this 0.4.0 to make it less confusing and at least clearify this is a later version than what is mentioned in the Readme.